### PR TITLE
update Req 1

### PR DESCRIPTION
**Pull request by Jaunius343**

### CODE REVIEW

File: 3aprasas.md [hunk 1]
```diff
@@ -89,20 +89,33 @@ User can have its favourite restaurants. In the page of a physical restaurant th
...
+   1.1. Add a new address
 
-Alternative scenario:
-If the address is not found on google maps – red “Address not found on google maps” modal shows up in the top right corner. User profile shows old addresses under “Addresses” subsection.
+   ![Add restaurant address use case](Assets/lab3/add-address.jpg)
 
+   Main scenario: 
+   Restaurant presses account bubble in website header. System loads restaurant information (Address, name, phone number) and opens restaurant editing page. Restaurant inputs new address in the address table. System finds the address with Google API and highlights a plus button with green color. User presses the plus button. System updates the address table.
```
[the remark this answers is not in the document]
Comment (Jaunius343) [reply 3aprasas.md:97]: Right, I was not sure about this one either. Based on the previous version, we used **Administrator** to describe  restaurant user, but restaurant is already included in the domain model as an attribute of user, that's why I thought that adding administrator attribute was excessive

**Comment by cezaris13** [comment]
Also, shouldn't the use case diagram be one for the requirement, instead of 2 for each use cace? @Mart1nux maybe you could help here?

**Comment by Jaunius343** [comment]
Based on the use cases we reviewed during the lecture, teacher specifically suggested that we split the use cases. We understood the concept of use cases wrong, and used alternatives as a way to describe different scenarios rather than commenting other variants to reach the same goal.

**Comment by Mart1nux** [comment]
Each bubble in the use case diagram that has a connection to the user should have its own use case. I think you could just have one use case diagram for one requirement and the have use case description for each of the bubble in the diagram after it. I'm not sure if it is required to separate them, especially when you end up with use case diagram that has a single use case.

**Comment by Jaunius343** [comment]
Oh, okay. It does make more sense, I'll merge my diagram then.

**Pull request merged**